org/wildfly/mod_cluster/capability.adoc

```diff
@@ -0,0 +1,19 @@
+NAME: org.wildfly.mod_cluster
```

Comment:
@rhusar It's fine to include private capabilities here. Not necessary to expose their service type or runtime API info, since no one outside can use them, but it's ok to include it too. The docs (see README.md) are clear about anything with PRIVATE: true not being usable externally and not providing any reliable external contract.

Including an entry here is mostly just a form of communication. What's already out there, what is using what (even if you are private, providers of other caps that you depend on may be interested), a bit of text describing what goes away if your private cap can't be installed because of a missing dependency. Not a ton of value to be sure, but OTOH there is some benefit to recording stuff.